docs/components/arm/_index.md

```diff
@@ -516,6 +516,64 @@ logger.Info(is_moving)
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetGeometries
+
+Get all the geometries associated with the arm in its current configuration, in the [frame](/services/frame-system/) of the arm.
+The [motion](/services/motion/) and [navigation](/services/navigation/) services use the relative position of inherent geometries to configured geometries representing obstacles for obstacle detection and avoidance while motion planning.
```

Comment:
@biotinker thoughts on accuracy of phrasing? Can change

Reply:
> obstacle detection and avoidance

"obstacle detection" is done elsewhere and is unrelated (for now) to `GetGeometries`. I would call this "collision detection and obstacle avoidance".

Other than that this sounds good.

Reply:
Fixed, thank you! @biotinker